packages/ibm-products-styles/src/components/NotificationsPanel/_notifications-panel.scss

```diff
@@ -270,6 +271,10 @@ $block-class: #{c4p-settings.$pkg-prefix}--notifications-panel;
     inset-block-end: 0;
     min-block-size: 2.5rem;
     .#{$block-class}__view-all-button {
+      display: flex;
+      width: 100%;
+      max-width: calc(100% - 2.5rem);
+      align-items: center;
```

Comment:
Fixes styling for "View all"

**Before**
![image](https://github.com/user-attachments/assets/b8e68d44-7a14-4467-b1b4-9cae1fdfc31b)

**After**
![image](https://github.com/user-attachments/assets/9cc74257-b445-4e5b-af54-5c7a9e7c8686)